Cython/Includes/cpython/list.pxd

```diff
@@ -1,5 +1,35 @@
 from .object cimport PyObject
 
+cdef extern from *:
+    # Backport PyList_GetItemRef, PyList_Extend and PyList_Clear
+    """
+    #if __PYX_LIMITED_VERSION_HEX < 0x030d0000
```

Comment:
`PyList_Extend` and `PyList_Clear` are not in the Limited API and thus need a different guard condition for `CYTHON_COMPILING_IN_LIMITED_API`.

Reply:
What should the behavior be? Should they be available? If that's the case, then this condition is okay, right?

Otherwise, if they shouldn't be, should an error be raised?

Reply:
Aah okay. I thought that `__PYX_LIMITED_VERSION_HEX` would cover this, but it doesn't for limited API above 3.13, right?